compiler/luci-micro/mbed-os/mbed-sources.cmake

```diff
@@ -0,0 +1,2560 @@
+macro(set_sources_mbed)
+    set(SOURCES
+            ${ARGV0}/cmsis/CMSIS_5/CMSIS/RTOS2/Include/cmsis_os2.h
```

Comment:
Q) are these code from outside?
Q) will they land in this repo?
Q) if they exist outside, how are they version controlled?

Reply:
As far as I know, this code is generated by [mbed-cli](https://github.com/ARMmbed/mbed-cli) tool, we need it for ARM MCU benchmark applications.

I asked @SlavikMIPT to write an instructions how to generate these files.
Using this instruction we can estimate how hard is it to call generator during CMake phase.

Reply:
because this code is generated, I answers are unclear =)
1. There are no particular place where we can get this code from outside
2. I hope not, but if generation process is not easy, this could happen =_=
3. -

Reply:
@SlavikMIPT ping?

Reply:
This is an edited generated with mbed-cli CMakeLists.txt
to generate CMakeLists.txt:
```
mbed import https://github.com/ARMmbed/mbed-os-example-blinky
cd mbed-os-example-blinky
cd mbed-os
pip install -r requirements.txt
cd ..
mbed export -i CMAKE_GCC_ARM --source . -m NUCLEO_F767ZI
```